src/components/Footer.tsx

```diff
@@ -1,13 +1,12 @@
 import React from "react";
 import Star from "./icons/Star";
-import { filteredResources } from "../data/data-resources";
 import { getColorByAccess } from "@/utils/nodes";
-
+const footerData = ["Open Source", "Institutional Access", "Paid Service"];
```

Comment:
Can you call this something else like "access" and put it data folder

Reply:
Fixed